crm/track_leads/lead_scoring.rst

```diff
@@ -20,59 +20,58 @@ To use scoring, install the free module *Lead Scoring* under your
 Create scoring rules
 ====================
 
-You now have a new tab in your *CRM* app called *Leads Management*
+You now have a new tab in your *CRM* app called *Leads*
 where you can manage your scoring rules.
 
-Here's an example for a Canadian lead, you can modify for whatever
-criteria you wish to score your leads on. You can add as many criterias
-as you wish.
+Here's an example of a Canadian lead. You can modify your rules for whatever
+criteria you want to score leads on. You can add as many criteria
+as you would like.
 
 .. image:: media/lead_scoring02.png
    :align: center
 
-Every hour every lead without a score will be automatically scanned and
-assigned their right score according to your scoring rules.
+Every lead without a score will automatically be scanned every hour and
```

Comment:
Generally, we write in the [present tense](https://www.odoo.com/documentation/user/14.0/contributing/documentation/content_guidelines.html#grammatical-tenses). Here there could be an argument to use the future tense, but "Every lead without a score is automatically scanned every hour" sounds good as well.